app/selenium_ui/confluence/pages/pages.py

```diff
@@ -83,6 +85,20 @@ def wait_for_comment_field(self):
     def click_edit(self):
         self.wait_until_clickable(PageLocators.edit_page_button).click()
 
+    def wait_for_resources_loaded(self):
+        sleep_time = 0.1
+        attempts = 30
+        total_sleep = 0
+
+        for _ in range(attempts):
+            loaded = self.execute_js("return require('confluence-editor-loader/editor-loader').resourcesLoaded();")
+            if loaded:
+                return
+            time.sleep(sleep_time)
+            total_sleep += sleep_time
+            print(f'Total sleep time: {total_sleep}')
```

Comment:
The message is not very clear.
Imagine that you see a message in the log: 
`Total sleep time: 2.3`